test/utils.tsx

```diff
@@ -76,13 +76,15 @@ export function validateCellPosition(columnIdx: number, rowIdx: number) {
 }
 
 export function copySelectedCell() {
+  // eslint-disable-next-line testing-library/prefer-user-event
```

Comment:
Is it worth exporting `fireEvent` from `testUtils` to avoid disabling this rule everywhere?

Reply:
I'd rather we did replace fireEvent whenever possible rather than allowing it, what's the point of enabling the rule otherwise